HackYourFuture-CPH/fp-class20
# Class20 72 product page container

# Description

ProductPage container that is used for assembling components to represent the product page

# How to test?

run Docker, connect db, run yarn dev, run storybook to see the result

https://hackyourfuture-dk.atlassian.net/browse/CLASS20-72?atlOrigin=eyJpIjoiNDIxNzIwMWI2YWI4NGQ3MGE4N2E2NDk5YTFkYjdmYmYiLCJwIjoiaiJ9

# Checklist
